services/storage/src/simcore_service_storage/redis.py

```diff
@@ -0,0 +1,35 @@
+import logging
+from typing import cast
+
+from aiohttp import web
+from servicelib.redis import RedisClientSDK
+from settings_library.redis import RedisDatabase, RedisSettings
+
+from .constants import APP_CONFIG_KEY
+from .settings import Settings
+
+_logger = logging.getLogger(__name__)
+
+_APP_REDIS_KEY = "APP_REDIS_KEY"
```

Comment:
Please rename like so as well
```suggestion
_APP_REDIS_MANAGER = "_APP_REDIS_MANAGER"
```